packages/@react-spectrum/button/src/Button.tsx

```diff
@@ -20,13 +20,18 @@ import {
 } from '@react-spectrum/utils';
 import {FocusableRef} from '@react-types/shared';
 import {FocusRing} from '@react-aria/focus';
+// @ts-ignore
+import intlMessages from '../intl/*.json';
 import {mergeProps} from '@react-aria/utils';
-import React, {ElementType, ReactElement} from 'react';
+import {ProgressCircle} from '@react-spectrum/progress';
+import React, {ElementType, ReactElement, useEffect, useRef, useState} from 'react';
 import {SpectrumButtonProps} from '@react-types/button';
+import {SpectrumProgressCircleProps} from '@react-types/progress';
 import styles from '@adobe/spectrum-css-temp/components/button/vars.css';
 import {Text} from '@react-spectrum/text';
 import {useButton} from '@react-aria/button';
 import {useHover} from '@react-aria/interactions';
+import {useLocalizedStringFormatter} from 'react-aria';
```

Comment:
We should use the individual package here (`@react-aria/i18n`) instead of the monopackage. That way if someone uses the individual button Spectrum package they don't need to install all of react aria.

Reply:
Good catch. I've pushed the change.